src/orquestra/integrations/qiskit/_get_provider.py

```diff
@@ -0,0 +1,26 @@
+from typing import Optional
+
+from qiskit_ibm_provider import IBMAccountError, IBMProvider  # type: ignore
+from qiskit_ibm_provider.api.exceptions import RequestsApiError  # type: ignore
```

Comment:
thought: Why is type: ignore needed here? Doesn't look like it was required before?

Reply:
suggesion: if `qiskit_ibm_provider` is not typed then you could avoid putting all these type: ignores individually by including `qiskit_ibm_provider` in the "ignore" section of the pyproject.toml.

Reply:
Good idea, implemented in ef01ccb